pkg/setting/settings.go

```diff
@@ -79,3 +80,61 @@ type ReloadHandler interface {
 	// fails the configuration will not be updated in the database
 	Validate(section Section) error
 }
+
+type SettingsBag map[string]map[string]string
+type SettingsRemovals map[string][]string
+
+type OSSImpl struct {
+	Cfg *Cfg `inject:""`
+}
+
+func (o OSSImpl) Init() error {
+	return nil
+}
+
+func (OSSImpl) Update(SettingsBag, SettingsRemovals) error {
+	return nil
```

Comment:
Should we return an error to give the feedback that updates are not supported (yet?)? 🤔

Reply:
Yeah, that seems like a good idea since this shouldn't ever be called.